DataCollector/RulerDataCollector.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Hoathis\Bundle\RulerBundle\DataCollector;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface;
+
+use Hoathis\Bundle\RulerBundle\Ruler\LoggedRuler;
```

Comment:
Well, that's an error. ping @jubianchi: Should we update the naming of the bench bundle?